src/test/java/messagerosa/xml/XMessageParserTest.java

```diff
@@ -1,22 +1,19 @@
 package messagerosa.xml;
 
-import messagerosa.core.model.XMessage;
 import org.junit.jupiter.api.Test;
 
-import javax.xml.bind.JAXBException;
 import java.io.FileInputStream;
-import java.io.FileNotFoundException;
+import java.io.IOException;
 import java.io.InputStream;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class XMessageParserTest {
 
     @Test
-    void parse() throws FileNotFoundException, JAXBException {
-        InputStream inputStream = new FileInputStream("~/message-rosa/src/test/java/messagerosa/xml/" + "testMessage.xml");
-        XMessage message = XMessageParser.parse(inputStream);
-        System.out.println(message.toXML());
-        assertEquals(1, 1);
+    void parse() throws IOException {
+		try (InputStream inputStream = new FileInputStream("src/test/java/messagerosa/xml/" + "testMessage.xml")) {
```

Comment:
Can we have them in the `resources` folder rather than `src` folder?

Reply:
Do you mean `src/main/resources` or a separate `resources` folder altogether?

Reply:
Ahh yes.